### PR TITLE
Change frogs to team

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- Use GH teams for frog ownership
**Related issue(s)**

### CODE REVIEW

**Comment by VOID404** [comment]
It looks like Frogs have individual write access to repo (required to own files), the team does not
/hold

**Review by a-thaler — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**
